base/services/tftpd/tftpd.cpp

```diff
@@ -2056,21 +2038,13 @@ void init(void *lpParam)
         }
     }
 
-    if ((f = openSection("ALLOWED-CLIENTS", 1, iniFile)))
+    if (f = openSection("ALLOWED-CLIENTS", 1, iniFile))
     {
-#ifdef __REACTOS__
-        MYWORD i = 0;
-#else
         int i = 0;
-#endif
 
         while (readSection(raw, f))
         {
-#ifdef __REACTOS__
-            if (i < _countof(cfig.hostRanges))
```

Comment:
I don't like how you replace that cfig.hostRanges with hardcodes. Also in other places with the MAX_SERVERS and the magic 8. Partly this and partly that is used. It's a total clusterfuck. This PR is far from being merge-ready.